ses

```diff
@@ -46,133 +46,156 @@ section_header "Ceph cluster status"
 
 mkdir $LOG/ceph
 
-if [ -x /usr/bin/ceph ]; then
-    ID=
-    if ! [ -f /etc/ceph/ceph.client.admin.keyring ]; then
-        # look for available keyring file and set client id accordingly
-        ID=$(ls /etc/ceph/ceph.client.*.keyring 2>/dev/null |
-                 awk -F. '{print "--id=" $3; exit}')
-    fi
+if [ -x /usr/sbin/cephadm ] ; then
+    CEPHADM_SHELL="/usr/sbin/cephadm shell --"
```

Comment:
could we do it more general? Like CEPH_SHELL ?
I could see we could add something like
`CEPH_SHELL="kubectl -n rook-ceph exec -it $(kubectl -n rook-ceph get pod -l "app=rook-ceph-tools" -o jsonpath='{.items[0].metadata.name}') --"`
which makes same thing for the Rook

Reply:
but we would need another var to identify what env we are working in.
`-x /usr/sbin/cephadm ` and etc might just become bool vars...

Reply:
Done in f025955 (there's no actual rook support yet, but at least the name is generic now, to support this).